doc/development.rst

```diff
@@ -314,43 +314,57 @@ Example
     # conftest.py
     from labgrid.strategy import GraphStrategy
 
-
     class TestStrategy(GraphStrategy):
-        def state_Unknown(self):
+        def state_unknown(self):
             pass
 
-        @GraphStrategy.depends('Unknown')
-        def state_Boot_via_NAND(self):
+        @GraphStrategy.depends('unknown')
+        def state_boot_via_nand(self):
             pass
 
-        @GraphStrategy.depends('Unknown')
-        def state_Boot_via_NFS(self):
+        @GraphStrategy.depends('unknown')
+        def state_boot_via_nfs(self):
             pass
 
-        @GraphStrategy.depends('Boot_via_NAND', 'Boot_via_NFS')
-        def state_BareBox(self):
+        @GraphStrategy.depends('boot_via_nand', 'boot_via_nfs')
+        def state_barebox(self):
             pass
 
-        @GraphStrategy.depends('BareBox')
-        def state_Linux_Shell(self):
+        @GraphStrategy.depends('barebox')
+        def state_linux_shell(self):
             pass
 
+The class can also render a graph as PNG (using GraphViz):
+
 .. code-block:: python
 
-    # render graph to png
-    >>> graph_strategy.graph.render('filename')
-    'filename.png'
+    >>> # use a test environment for this example
+    >>> import labgrid
+    >>> env = labgrid.environment.Environment('myplace.yaml')
```

Comment:
Can you add the example 'myplace.yaml' to the documentation as well?

Reply:
Right, good idea. Updated the docs once more.